unitTests/CMakeLists.txt

```diff
@@ -22,7 +22,6 @@ target_link_libraries(unit_tests
     PRIVATE
         exiv2lib
         GTest::GTest
-        GTest::Main
```

Comment:
Good to know. I used GTest for years, and I never took the time to check whether gtest_main was needed or not.

Reply:
You can use `GTest::Main`, then you don't have to provide a `main()` that runs all tests yourself. But that requires that `gtest_main.a/.so` exists, which it doesn't on Alpine, so I'd prefer to keep it the way it is.

Reply:
Sure. What I meant is that I always include it without thinking what it was. I actually always provide my own main function, specially in big projects where you need to do advance stuff ;)